### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by Foxcapades**

### CODE REVIEW

File: schema/components/reverse.raml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+  ReverseRequest:
```
Comment (steve-fischer-200) [inline schema/components/reverse.raml:5]: a comment to explain what a ReverseRequest is would help.  not something i can guess from here

Comment (steve-fischer-200) [reply schema/components/reverse.raml:5]: nevermind...

**Review by steve-fischer-200 — APPROVED** [review]

File: readme.adoc [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+----
+[
+  "output"
+]
+----
+
+
+==== Job File Download
+
+The outputs of a completed job can
```
Comment (dmgaldi) [inline readme.adoc:123]: Incomplete thought

**Review by dmgaldi — APPROVED** [review]

File: readme.adoc (hunk 2 above)
Comment (Foxcapades) [reply readme.adoc:123]: Thanks, I got sidetracked.

**Review by steve-fischer-200 — APPROVED** [review]
requested re-review by accident

**Pull request merged**